recipes/libhal/all/conandata.yml

```diff
@@ -2,3 +2,6 @@ sources:
   "2.0.1":
     url: "https://github.com/libhal/libhal/archive/refs/tags/2.0.1.tar.gz"
     sha256: "ba05f9d2172f7afbf816ab18f2dcd7c8cc4452df2b268a832e995a0321c1d2f9"
+  "2.0.2":
```

Comment:
Could you put the newer version above? We try to keep the versions ordered in that way.